config-bot/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:30
+FROM quay.io/coreos-assembler/coreos-assembler:master
```

Comment:
Hmm. Things just got a lot more heavyweight :) - Does this mean we also require /dev/kvm for config-bot now?

Reply:
I originally wanted to use cosa for this, and had a patch to drop the req. on `/dev/kvm` for our needs. :) But in the end, I think it's simpler to just invoke `rpm-ostree` ourselves directly.